api/tests/unit/features/test_unit_features_models.py

```diff
@@ -220,322 +218,295 @@ def test_when_create_feature_with_tags_then_success(project: Project) -> None:
-        )
-        self.environment = Environment.objects.create(
-            name="Test environment", project=self.project
-        )
-        self.feature = Feature.objects.create(name="Test feature", project=self.project)
-
-    @mock.patch("features.signals.trigger_feature_state_change_webhooks")
-    def test_cannot_create_duplicate_feature_state_in_an_environment(
-        self, mock_trigger_webhooks
-    ):
-        """
-        Note that although the mock isn't used in this test, it throws an exception on
-        it's thread so we mock it here anyway.
-        """
-
-        # Given
-        duplicate_feature_state = FeatureState(
-            feature=self.feature, environment=self.environment, enabled=True
-        )
+@mock.patch("features.signals.trigger_feature_state_change_webhooks")
+def test_cannot_create_duplicate_feature_state_in_an_environment(
+    mock_trigger_webhooks: mock.MagicMock,
+    feature: Feature,
+    environment: Environment,
+) -> None:
+    """
+    Note that although the mock isn't used in this test, it throws an exception on
+    it's thread so we mock it here anyway.
+    """
 
-        # When
-        with pytest.raises(ValidationError):
-            duplicate_feature_state.save()
+    # Given
+    duplicate_feature_state = FeatureState(
+        feature=feature, environment=environment, enabled=True
+    )
 
-        # Then
-        assert (
-            FeatureState.objects.filter(
-                feature=self.feature, environment=self.environment
-            ).count()
-            == 1
-        )
+    # When
+    with pytest.raises(ValidationError):
+        duplicate_feature_state.save()
 
-    @mock.patch("features.signals.trigger_feature_state_change_webhooks")
-    def test_cannot_create_duplicate_feature_state_in_an_environment_for_segment(
-        self, mock_trigger_webhooks
-    ):
-        """
-        Note that although the mock isn't used in this test, it throws an exception on
-        it's thread so we mock it here anyway.
-        """
-
-        # Given
-        segment = Segment.objects.create(project=self.project)
-        feature_segment = FeatureSegment.objects.create(
-            feature=self.feature, environment=self.environment, segment=segment
-        )
-        FeatureState.objects.create(
-            feature=self.feature,
-            environment=self.environment,
-            feature_segment=feature_segment,
-        )
+    # Then
+    assert (
+        FeatureState.objects.filter(feature=feature, environment=environment).count()
+        == 1
+    )
+
+
+@mock.patch("features.signals.trigger_feature_state_change_webhooks")
+def test_cannot_create_duplicate_feature_state_in_an_environment_for_segment(
+    mock_trigger_webhooks: mock.MagicMock,
+    project: Project,
+    feature: Feature,
+    environment: Environment,
+) -> None:
+    """
+    Note that although the mock isn't used in this test, it throws an exception on
+    it's thread so we mock it here anyway.
```

Comment:
I don't understand this comment.

Reply:
I removed the mock and it looks like it passes without it, so I'll just delete this docstring and update the code. Thanks for checking it!

Reply:
(Note I'm also removing the other similar test mocks in the module.)